app/Post_Types/Archive_Item/Post_Meta.php

```diff
@@ -146,6 +146,19 @@ public function add_dynamic_fields(){
 
 			}
 		}
+
+		// add the related field
+		$related = carbon_get_theme_option(\Diviner\Admin\Settings::FIELD_GENERAL_RELATED_FIELD);
+		if ( $related ) {
+			$field = \Diviner\Post_Types\Diviner_Field\Types\Related_Field::render(
```

Comment:
all these fully-qualified references, `\Acme\Company\Package\SubPackage\EtCetera` could be simplified using `use`; this line could be rewritten like:

```php
use \Diviner\Post_Types\Diviner_Field\Types\Related_Field;

//...

RelatedField::render(...)
```

Reply:
Done